packages/x-charts/src/ChartsTooltip/ChartsTooltip.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { unstable_composeClasses as composeClasses } from '@mui/utils';
-import Popper from '@mui/material/Popper';
-import NoSsr from '@mui/material/NoSsr';
+import { Popper } from '@mui/base/Popper';
+import { NoSsr } from '@mui/base/NoSsr';
```

Comment:
Looks great. However, we miss `@mui/base` in the package.json.

Reply:
Seems it's a topi common to all X libraries. So I merge this PR as it, and will do two follow up:
- The base popup experiment
- Add base/system in the dependencies depending on how the all X evolves

Bellow the slack message about @mui/system deps
---

Good morning everyone,
I've started a project from scratch today and noticed the following warnings:
```bash
warning " > @mui/x-charts@6.0.0-alpha.11" has unmet peer dependency "@mui/system@^5.4.1".
warning " > @mui/x-data-grid-premium@6.14.0" has unmet peer dependency "@mui/system@^5.4.1".
warning "@mui/x-data-grid-premium > @mui/x-data-grid@6.14.0" has unmet peer dependency "@mui/system@^5.4.1".
warning "@mui/x-data-grid-premium > @mui/x-data-grid-pro@6.14.0" has unmet peer dependency "@mui/system@^5.4.1".
warning " > @mui/x-tree-view@6.0.0-alpha.3" has unmet peer dependency "@mui/system@^5.8.0".
warning "@mui/x-date-pickers-pro > @mui/x-date-pickers@6.14.0" has unmet peer dependency "@mui/system@^5.8.0".
warning " > @mui/x-date-pickers-pro@6.14.0" has unmet peer dependency "@mui/system@^5.8.0".
```
We should (probably) mention @mui/system on the install guides. But also, why are the versions inconsistent?